src/index.ts

```diff
@@ -72,6 +73,14 @@ router.post(
   loadActiveSurveyForDao,
   submitContribution
 )
+// Nominate contribution.
+router.post(
+  '/:dao/nominate',
+  loadDaoFromParams,
+  loadActiveSurveyForDao,
+  authDaoMemberAtSurveyCreationBlockHeightMiddleware,
```

Comment:
Incredible name, 10/10. Long, but 0 ambiguity about what this is and what it does.